docs/Developers/Git_Basics.rst

```diff
@@ -283,6 +283,54 @@ another editor will be fired up with all the messages of commits which you want
 to squash. Clean up all the messages and add a new message to be
 displayed for the single commit.
 
+Common Git Issues
+-----------------
+Sometimes, it might happen that after rebasing, you use ``git add -A`` and
```

Comment:
I would change this to -> "Sometimes, you use git ``add-A`` and add files you didn't want to your push (often after rebasing)"